sdk/textanalytics/azure-ai-textanalytics/CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 # Release History
 ## 5.1.0-beta.5 (Unreleased)
-
+### Breaking changes
+- Renamed `AspectSentiment` to `TargetSentiment`, `OpinionSentiment` to `AssesssmentSentiment`.
```

Comment:
As part of the renames, need to update https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/textanalytics/azure-ai-textanalytics#analyze-sentiment

Reply:
still missing

Reply:
Do you mean this sentence, "To get more granular information about the opinions related to aspects of a product/service, also knows as Aspect-based Sentiment Analysis in Natural Language Processing (NLP), see sample on sentiment analysis with opinion mining see here."?

Reply:
Hmm you are right, no need to change anything here. Sorry about that